todolist/settings.py

```diff
@@ -21,9 +21,9 @@
 SECRET_KEY = "@e2(yx)v&tgh3_s=0yja-i!dpebxsz^dg47x)-k&kq_3zf*9e*"
```

Comment:
Storing the secret key directly in the `settings.py` file is not secure. Consider using environment variables or a separate configuration file to manage sensitive information securely.